.evergreen/evergreen.yml.in

```diff
@@ -1112,7 +1112,7 @@ tasks:
   ###
   <% for (const { id, nShort, nVersion, mShort, mVersion, skipNodeVersionCheck, packageName } of ALL_UNIT_TESTS) { %>
   - name: test_<% out(id) %>
-    tags: <% out(["unit-test", ...(mShort === 'latest' ? ["mlatest"] : [])]) %>
+    tags: <% out(["assigned_to_jira_team_mongosh_mongosh", "unit-test", ...(mShort === 'latest' ? ["mlatest"] : [])]) %>
```

Comment:
there's also `foliage_check_task_only` which will create tickets for the whole task instead of a specific test failure but I think we should be fine regardless?

only worry I have with this is that because of our setup we might end up with a spam of test failures for a task on every platform if 1 test fails

Reply:
Hm yeah, I can see your concern ... but our task names already include platform names, I guess making it a per-task thing wouldn't actually help all that much?

Reply:
yeah this is basically unavoidable without refactoring our tasks a lot more, we could just put it in and see what happens I suppose